setup.py

```diff
@@ -36,7 +36,7 @@ def read(filename):
       zip_safe=False,
       install_requires=[
           'setuptools',
-          # -*- Extra requirements: -*-
+          'colorama',
```

Comment:
Note that `wheel` is currently only a `recommended` dependency. We could promote it a real dependency. But I guess that is not needed, for example when you use zest.releaser to release a non-python project.